examples/object_detection/layers/extensions/nms/nms.cpp

```diff
@@ -4,8 +4,5 @@
 #include "nms.h"
 
 at::Tensor nms(const at::Tensor& boxes, const float nms_overlap_thresh, const int64_t top_k) {
-  if (boxes.type().is_cuda()) {
-    return nms_gpu(boxes, nms_overlap_thresh, top_k);
-  }
-  return nms_cpu(boxes, nms_overlap_thresh, top_k);
+  return nms_gpu(boxes, nms_overlap_thresh, top_k);
```

Comment:
Do we need this empty wrapper anymore? Why not just leave the `nms_gpu` function?